package.json

```
@@ -9,7 +9,7 @@
     "lint": "next lint"
   },
   "devDependencies": {
-    "@ant-design/cssinjs": "^1.3.0",
+    "@ant-design/cssinjs": "~1.8.1",
```

```suggestion
    "@ant-design/cssinjs": "^1.8.1",
```
antd 的依赖是 `^` ，这里尽量一致。